core/tests/test_utils.py

```diff
@@ -1773,7 +1773,14 @@ def setUp(self):
         self.taskqueue_testapp = webtest.TestApp(main_taskqueue.app)
         self.mail_testapp = webtest.TestApp(main_mail.app)
 
-        self.signup_superadmin_user()
+        # NOTE: For tests that do not/can not use the default superadmin,
+        # authors can the class-level constant:
```

Comment:
*can set

Also, I think the usual way we do this is not using getattr. Instead, define a class-level constant and set it to True in the base class. Subclasses can override. Then do a check on self.SIGNUP_DEFAULT_SUPERADMIN_USER directly.

Maybe self.AUTO_CREATE_DEFAULT_SUPERADMIN_USER is better too, so it does not start with a verb.

Reply:
Done.